docs/conf.py.in

```diff
@@ -42,7 +45,8 @@ extensions = [
     'sphinxcontrib.manpage',
     'sphinxcontrib.blockdiag',
     'sphinxcontrib.seqdiag',
-    'sphinxcontrib.actdiag'
+    'sphinxcontrib.actdiag',
+    'taglist'
```

Comment:
This should probably be an extension we want to upstream, if it isn't already available anywhere, right? Could actually be worth putting it in its own repo eventually.

Reply:
Could be. It may help to do so as repositories using the blueprint currently does not inherit the blueprint configuration automatically.

Reply:
Please approve or add further comments if you want further changes.

Reply:
I just meant that the taglist extension maybe should be put in its own repo completely later on, if we want to upstream it. But that is a later issue anyway.